### ISSUE TITLE
Howto or instuction how to install Reverb pulseaudio voluem control on Android

### ISSUE THREAD

**Issue by MurzNN** [hchapman/reverb#1]
Idea of Reverb application is very good - I very long time searching for application that can remotely control of pulseaudio volume from android system and find nothing. But now I find your app, via screenshots it looks very good. But I don't understand how to install it on my android system.

Can you describe instructions how to install your app on android phone?

**Comment by MurzNN** [comment]
Reverb APK file from link https://github.com/hchapman/reverb/Reverb.apk/qr_code is removed now. Can you give it back? It installs and works normally on my android phone, but after reinstall system I lost APK file and can't find where can I download it.
I know that this is too old version, but this is better than nothing, because I can't find any other app or way to control pulseaudio volume remotely via Android.

**Comment by hchapman** [comment]
"New" link is https://github.com/downloads/hchapman/reverb/Reverb.apk
Version is unknown, but should be the same. This is probably tough because github is/was phasing out their downloads feature.

**Comment by parkerlreed** [comment]
People seem to use the releases section for downloads https://github.com/hchapman/reverb/releases

Might be good to get it going there at least.

**Comment by hchapman** [comment]
Yes I'll look into it over the course of the next week, especially if I can
find that old APK
